android/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/ScanManager.java

```diff
@@ -121,24 +130,36 @@ public synchronized void startScanning(Context context) {
         mIsMotionlessPausedState = false;
 
         if (mLocationChangeSensor == null) {
-            mLocationChangeSensor = new LocationChangeSensor(mContext, mDetectUserIdleReceiver);
+            mLocationChangeSensor = new LocationChangeSensor(mAppContext, mDetectUserIdleReceiver);
         }
         mLocationChangeSensor.start();
 
         if (mMotionSensor == null) {
-            mMotionSensor = new MotionSensor(mContext, mDetectMotionReceiver);
+            mMotionSensor = new MotionSensor(mAppContext, mDetectMotionReceiver);
         }
 
         if (mGPSScanner == null) {
-            mGPSScanner = new GPSScanner(context, this);
-            mWifiScanner = new WifiScanner(context);
-            mCellScanner = new CellScanner(context);
+            mGPSScanner = new GPSScanner(mAppContext, this);
+            mWifiScanner = new WifiScanner(mAppContext);
+            mCellScanner = new CellScanner(mAppContext);
```

Comment:
@crankycoder just a reminder that you wanted to take a look at removing this presumably obsolete code block